internal/services/dashboard/dashboard_grafana_resource_test.go

```diff
@@ -178,9 +179,10 @@ func (r DashboardGrafanaResource) update(data acceptance.TestData) string {
 			%s
 
 resource "azurerm_dashboard_grafana" "test" {
-  name                = "a-dg-%d"
-  resource_group_name = azurerm_resource_group.test.name
-  location            = azurerm_resource_group.test.location
+  name                  = "a-dg-%d"
+  resource_group_name   = azurerm_resource_group.test.name
+  location              = azurerm_resource_group.test.location
+  grafana_major_version = "10"
```

Comment:
This field is `ForceNew` meaning setting this in the Update test would actually create a new resource, which is problematic. Instead, can we add a new acceptance test covering provisioning with v10?

Reply:
@tombuildsstuff I can see your point here, however, I do think it's a good test to reprovision. Only real effect is time taken for the tests. I'm also fine to write a new test creating a v10 resource, I only worry that the available versions will grow over time and whether or not it makes sense to have a test for each version (unless I'm missing your point).